src/index.js

```diff
@@ -31,7 +35,14 @@ function startElectron (config, ports) {
         args = debugPortsArgs.concat(extraArgs);
     }
 
-    spawn(cmd, args, { stdio: 'ignore' });
+    var proc = spawn(cmd, args, { stdio: ['ignore', 'pipe', 'pipe'] });
+
+    proc.stdout.on('data', (buf) => {
+        STDOUT_LOGGER(buf.toString().trim());
```

Comment:
It comes from my personal experience, but I try to avoid processing debug output. I know, the output can be a bit ugly without `trim()`, but simplicity is better than beauty for debug logs. 

Also, `String(value)` can handle primitive values like numbers, `null` and `undefined` without exceptions, so it's better to use it for debugging since in JS world you get literally anything as a result and we don't want to ruin the app's process with an exception in the debug logger.

Reply:
the `buf` value is part of the node core `child_process` API and its a `Buffer` ; converting it to a string should really be `buf.toString('utf8')` but `String(buf)` does work as well.

As for trim; I want to trim a trailing new line because the output has a trailing new line and the `debug` module also prints a new line.

This leads to double new line output; the same problem would happen if you did `console.log(buf.toString('utf8'))` instead of `process.stdout.write(buf)` ;

console.log adds a new line for you, `process.stdout.write` just writes buffers to stdout and your responsible for new lines.